src/kernels/variables/debuggerVariables.ts

```diff
@@ -9,9 +9,13 @@ import { DebugAdapterTracker, Disposable, Event, EventEmitter } from 'vscode';
 import { DebugProtocol } from 'vscode-debugprotocol';
 import { Identifiers } from '../../platform/common/constants';
 import { IDebugService, IVSCodeNotebook } from '../../platform/common/application/types';
-import { DataFrameLoading, GetVariableInfo } from '../../platform/common/scriptConstants';
 import { traceError, traceVerbose } from '../../platform/logging';
-import { IConfigurationService, IExtensionContext, Resource } from '../../platform/common/types';
+import {
```

Comment:
Note: I haven't been able to repro the issue yet, however I can see where it can fail.
Will try to repro this (having trouble debugging IW right now)

Reply:
There's a repro in the bug. You don't need to use IW to repro. Just debug a plain python file.

Reply:
aaah got it, thanks